### PR TITLE
Remove SlotsCrowdloanIndexMigration & InjectValidatorsIntoVoterList

### PR DESCRIPTION
**Pull request by coderobe**

### CODE REVIEW

**Comment by coderobe** [comment]
i'll also pick this in a follow-up PR against master if it passes review here

File: runtime/common/src/crowdloan/mod.rs [hunk 1]
```diff
@@ -49,8 +49,6 @@
 //! the parachain remains active. Users can withdraw their funds once the slot is completed and funds are
 //! returned to the crowdloan account.
 
-pub mod migration;
-
```
Comment (shawntabrizi) [inline runtime/common/src/crowdloan/mod.rs:53]: Why remove this?

Seems fine to expose this module, and keep the migration above.

File: runtime/kusama/src/lib.rs [hunk 2]
```diff
@@ -1531,31 +1530,11 @@ pub type Executive = frame_executive::Executive<
...
-	fn pre_upgrade() -> Result<(), &'static str> {
-		slots::migration::slots_crowdloan_index_migration::pre_migrate::<Runtime>()
-	}
-
-	#[cfg(feature = "try-runtime")]
-	fn post_upgrade() -> Result<(), &'static str> {
-		slots::migration::slots_crowdloan_index_migration::post_migrate::<Runtime>()
-	}
-}
-
```
Comment (shawntabrizi) [inline runtime/kusama/src/lib.rs:1558]: These are the only things that need to be removed.

**Comment by shawntabrizi** [comment]
No need to remove the migration code in general, just remove its use.

File: runtime/common/src/crowdloan/mod.rs (hunk 1 above)
Comment (coderobe) [reply runtime/common/src/crowdloan/mod.rs]: seemed sensible to remove the dead code together with its callsites
anyway, reverted

File: runtime/kusama/src/lib.rs [hunk 3]
```diff
@@ -1531,31 +1530,11 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	(
-		SlotsCrowdloanIndexMigration,
-		pallet_staking::migrations::v9::InjectValidatorsIntoVoterList<Runtime>,
-	),
+	(pallet_staking::migrations::v9::InjectValidatorsIntoVoterList<Runtime>,),
```
Comment (dvdplm) [inline runtime/kusama/src/lib.rs:1533]: So this should be removed as well? (says [Kian here](https://github.com/paritytech/polkadot/issues/5354#issuecomment-1108184161))

File: runtime/polkadot/src/lib.rs [hunk 4]
```diff
@@ -1484,32 +1483,11 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	(
-		SlotsCrowdloanIndexMigration,
-		pallet_staking::migrations::v9::InjectValidatorsIntoVoterList<Runtime>,
-	),
+	(pallet_staking::migrations::v9::InjectValidatorsIntoVoterList<Runtime>,),
```
Comment (dvdplm) [inline runtime/polkadot/src/lib.rs:1486]: Same: should be removed yes?

File: runtime/westend/src/lib.rs [hunk 5]
```diff
@@ -1093,32 +1093,11 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	(
-		SlotsCrowdloanIndexMigration,
-		pallet_staking::migrations::v9::InjectValidatorsIntoVoterList<Runtime>,
-	),
+	(pallet_staking::migrations::v9::InjectValidatorsIntoVoterList<Runtime>,),
```
Comment (dvdplm) [inline runtime/westend/src/lib.rs:1096]: Same?

File: runtime/kusama/src/lib.rs (hunk 3 above)
Comment (coderobe) [reply runtime/kusama/src/lib.rs:1533]: Indeed, i have a follow-up PR lined up. I can also pull that change in here if you'd like, though

Comment (dvdplm) [reply runtime/kusama/src/lib.rs:1533]: Yeah, let's keep them together.

Comment (coderobe) [reply runtime/kusama/src/lib.rs]: ok

**Review by dvdplm — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Review by shawntabrizi — COMMENTED** [review]
looks good, thank you

**Pull request merged**